lib/iris/tests/integration/test_pp.py

```diff
@@ -587,5 +588,56 @@ def test_as_pairs(self):
             self.assertEqual(cube.attributes['pseudo level'], 3)
 
 
+class TestSaveLBPROC(tests.IrisTest):
+
+    def create_cube(self):
+        cube = Cube(np.zeros((2, 3, 4)))
+        cube.add_dim_coord(
+            DimCoord(standard_name='time',
+                     units=Unit('days since 0001-01-01 00:00:00',
+                                calendar='gregorian'),
+                     points=np.arange(2),),
+            0)
+        cube.add_dim_coord(DimCoord(standard_name='longitude',
+                                    units='degrees',
+                                    points=np.arange(3),), 1)
```

Comment:
No problem, and thanks for clarifying.  The reason for choosing standard_name was almost certainly the result of a copy and paste from an existing test, for what it's worth, so I wouldn't read too much into it.  That would explain why the redundant latitude name was still present though.